src/gui/SelectableRowContainer.ts

```diff
@@ -51,14 +60,20 @@ export class SelectableRowContainer implements ClassComponent<SelectableRowConta
 		)
 	}
 
+	private setBackground(color: string) {
+		if (this.dom) this.dom.style.backgroundColor = color
+	}
+
+	private static getIsUsingKeyboard() {
```

Comment:
could rename it to sound more natural

Reply:
Fair enough. How about `getIfUsingKeyboard()`?

Reply:
I would suggest `isUsingKeyboard()`